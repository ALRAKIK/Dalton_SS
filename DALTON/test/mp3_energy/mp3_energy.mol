BASIS
6-31G
Test system for CPS(D) energy series.

Atomtypes=1 Nosymmetry Angstrom Charge=-1
Charge=9 Atoms=1
F   0.000000   0.000000        0.000000
