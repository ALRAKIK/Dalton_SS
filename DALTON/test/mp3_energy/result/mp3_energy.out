

     ************************************************************************
     *************** Dalton - An Electronic Structure Program ***************
     ************************************************************************

    This is output from DALTON release Dalton2021.alpha (2020)
         ( Web site: http://daltonprogram.org )

   ----------------------------------------------------------------------------

    NOTE:
     
    Dalton is an experimental code for the evaluation of molecular
    properties using (MC)SCF, DFT, CI, and CC wave functions.
    The authors accept no responsibility for the performance of
    the code or for the correctness of the results.
     
    The code (in whole or part) is provided under a licence and
    is not to be reproduced for further distribution without
    the written permission of the authors or their representatives.
     
    See the home page "http://daltonprogram.org" for further information.
     
    If results obtained with this code are published,
    the appropriate citations would be both of:
     
       K. Aidas, C. Angeli, K. L. Bak, V. Bakken, R. Bast,
       L. Boman, O. Christiansen, R. Cimiraglia, S. Coriani,
       P. Dahle, E. K. Dalskov, U. Ekstroem,
       T. Enevoldsen, J. J. Eriksen, P. Ettenhuber, B. Fernandez,
       L. Ferrighi, H. Fliegl, L. Frediani, K. Hald, A. Halkier,
       C. Haettig, H. Heiberg, T. Helgaker, A. C. Hennum,
       H. Hettema, E. Hjertenaes, S. Hoest, I.-M. Hoeyvik,
       M. F. Iozzi, B. Jansik, H. J. Aa. Jensen, D. Jonsson,
       P. Joergensen, J. Kauczor, S. Kirpekar,
       T. Kjaergaard, W. Klopper, S. Knecht, R. Kobayashi, H. Koch,
       J. Kongsted, A. Krapp, K. Kristensen, A. Ligabue,
       O. B. Lutnaes, J. I. Melo, K. V. Mikkelsen, R. H. Myhre,
       C. Neiss, C. B. Nielsen, P. Norman, J. Olsen,
       J. M. H. Olsen, A. Osted, M. J. Packer, F. Pawlowski,
       T. B. Pedersen, P. F. Provasi, S. Reine, Z. Rinkevicius,
       T. A. Ruden, K. Ruud, V. Rybkin, P. Salek, C. C. M. Samson,
       A. Sanchez de Meras, T. Saue, S. P. A. Sauer,
       B. Schimmelpfennig, K. Sneskov, A. H. Steindal,
       K. O. Sylvester-Hvid, P. R. Taylor, A. M. Teale,
       E. I. Tellgren, D. P. Tew, A. J. Thorvaldsen, L. Thoegersen,
       O. Vahtras, M. A. Watson, D. J. D. Wilson, M. Ziolkowski
       and H. Agren,
       "The Dalton quantum chemistry program system",
       WIREs Comput. Mol. Sci. 2014, 4:269â€“284 (doi: 10.1002/wcms.1172)
     
    and
     
       Dalton, a molecular electronic structure program,
       Release Dalton2021.alpha (2020), see http://daltonprogram.org
   ----------------------------------------------------------------------------

    Authors in alphabetical order (Major contribution(s) in parenthesis):

  Kestutis Aidas,           Vilnius University,           Lithuania   (QM/MM)
  Celestino Angeli,         University of Ferrara,        Italy       (NEVPT2)
  Keld L. Bak,              UNI-C,                        Denmark     (AOSOPPA, non-adiabatic coupling, magnetic properties)
  Vebjoern Bakken,          University of Oslo,           Norway      (DALTON; geometry optimizer, symmetry detection)
  Radovan Bast,             UiT The Arctic U. of Norway,  Norway      (DALTON installation and execution frameworks)
  Pablo Baudin,             University of Valencia,       Spain       (Cholesky excitation energies)
  Linus Boman,              NTNU,                         Norway      (Cholesky decomposition and subsystems)
  Bruno N. Cabral Tenorio,  Technical Univ. of Denmark,   Denmark     (Lanczos-based CC methods)
  Ove Christiansen,         Aarhus University,            Denmark     (CC module)
  Renzo Cimiraglia,         University of Ferrara,        Italy       (NEVPT2)
  Sonia Coriani,            Technical Univ. of Denmark,   Denmark     (CC and RESPONS modules)
  Janusz Cukras,            University of Warsaw,         Poland      (MChD in RESPONS)
  Paal Dahle,               University of Oslo,           Norway      (Parallelization)
  Erik K. Dalskov,          UNI-C,                        Denmark     (SOPPA)
  Thomas Enevoldsen,        Univ. of Southern Denmark,    Denmark     (SOPPA)
  Janus J. Eriksen,         Aarhus University,            Denmark     (Polarizable embedding, TDA)
  Rasmus Faber,             Technical Univ. of Denmark,   Denmark     (Vib.avg. NMR w. SOPPA, parallel AO-SOPPA, CVS-CC and EOMCC)
  Tobias Fahleson,          KTH Stockholm,                Sweden      (Damped cubic response)
  Berta Fernandez,          U. of Santiago de Compostela, Spain       (Doublet spin, ESR in RESPONS)
  Lara Ferrighi,            Aarhus University,            Denmark     (PCM Cubic response)
  Heike Fliegl,             University of Oslo,           Norway      (CCSD(R12))
  Luca Frediani,            UiT The Arctic U. of Norway,  Norway      (PCM)
  Emmanuel Fromager,        Univ. Strasbourg,             France      (MC-srDFT)
  Bin Gao,                  UiT The Arctic U. of Norway,  Norway      (Gen1Int library)
  Andre S. P. Gomes,        CNRS/Universite de Lille,     France      (Frozen density embedding)
  Christof Haettig,         Ruhr-University Bochum,       Germany     (CC module)
  Kasper Hald,              Aarhus University,            Denmark     (CC module)
  Asger Halkier,            Aarhus University,            Denmark     (CC module)
  Frederik Beyer Hansen,    University of Copenhagen,     Denmark     (Parallel AO-SOPPA)
  Erik D. Hedegaard,        Univ. of Southern Denmark,    Denmark     (Polarizable embedding, QM/MM, and MC-srDFT)
  Hanne Heiberg,            University of Oslo,           Norway      (Geometry analysis, selected one-electron integrals)
  Trygve Helgaker,          University of Oslo,           Norway      (DALTON; ABACUS, ERI, DFT modules, London, and much more)
  Alf Christian Hennum,     University of Oslo,           Norway      (Parity violation)
  Hinne Hettema,            University of Auckland,       New Zealand (Quadratic response in RESPONS; SIRIUS supersymmetry)
  Eirik Hjertenaes,         NTNU,                         Norway      (Cholesky decomposition)
  Pi A. B. Haase,           University of Copenhagen,     Denmark     (Triplet AO-SOPPA)
  Maria Francesca Iozzi,    University of Oslo,           Norway      (RPA)
  Christoph Jacob           TU Braunschweig               Germany     (Frozen density embedding)
  Brano Jansik              Technical Univ. of Ostrava    Czech Rep.  (DFT cubic response)
  Hans Joergen Aa. Jensen,  Univ. of Southern Denmark,    Denmark     (DALTON; SIRIUS, RESPONS, ABACUS, MC-srDFT, and much more)
  Dan Jonsson,              UiT The Arctic U. of Norway,  Norway      (Cubic response in RESPONS module)
  Poul Joergensen,          Aarhus University,            Denmark     (RESPONS, ABACUS, and CC modules)
  Maciej Kaminski,          University of Warsaw,         Poland      (CPPh in RESPONS)
  Joanna Kauczor,           Linkoeping University,        Sweden      (Complex polarization propagator (CPP) module)
  Sheela Kirpekar,          Univ. of Southern Denmark,    Denmark     (Mass-velocity & Darwin integrals)
  Erik Kjellgren,           Univ. of Southern Denmark,    Denmark     (MC-srDFT)
  Wim Klopper,              KIT Karlsruhe,                Germany     (R12 code in CC, SIRIUS, and ABACUS modules)
  Stefan Knecht,            ETH Zurich,                   Switzerland (Parallel CI and MCSCF, MC-srDFT)
  Rika Kobayashi,           Australian National Univ.,    Australia   (DIIS in CC, London in MCSCF)
  Henrik Koch,              NTNU,                         Norway      (CC module, Cholesky decomposition)
  Jacob Kongsted,           Univ. of Southern Denmark,    Denmark     (Polarizable embedding, polarizable density embedding, QM/MM)
  Andrea Ligabue,           University of Modena,         Italy       (CTOCD, AOSOPPA)
  Nanna H. List             Univ. of Southern Denmark,    Denmark     (Polarizable embedding)
  Ola B. Lutnaes,           University of Oslo,           Norway      (DFT Hessian)
  Juan I. Melo,             University of Buenos Aires,   Argentina   (LRESC, Relativistic Effects on NMR Shieldings)
  Kurt V. Mikkelsen,        University of Copenhagen,     Denmark     (MC-SCRF and QM/MM)
  Rolf H. Myhre,            NTNU,                         Norway      (Subsystems and CC3)
  Christian Neiss,          Univ. Erlangen-Nuernberg,     Germany     (CCSD(R12))
  Christian B. Nielsen,     University of Copenhagen,     Denmark     (QM/MM)
  Patrick Norman,           KTH Stockholm,                Sweden      (Cubic response and complex frequency response in RESPONS)
  Jeppe Olsen,              Aarhus University,            Denmark     (SIRIUS CI/density modules)
  Jogvan Magnus H. Olsen,   Aarhus University,            Denmark     (Polarizable embedding and polarizable density embedding)
  Anders Osted,             Copenhagen University,        Denmark     (QM/MM)
  Martin J. Packer,         University of Sheffield,      UK          (SOPPA)
  Filip Pawlowski,          Kazimierz Wielki University,  Poland      (CC3)
  Morten N. Pedersen,       Univ. of Southern Denmark,    Denmark     (Polarizable embedding)
  Thomas B. Pedersen,       University of Oslo,           Norway      (Cholesky decomposition)
  Patricio F. Provasi,      University of Northeastern,   Argentina   (Analysis of coupling constants in localized orbitals)
  Peter Reinholdt,          Univ. of Southern Denmark,    Denmark     (Polarizable embedding and polarizable density embedding)
  Zilvinas Rinkevicius,     KTH Stockholm,                Sweden      (open-shell DFT, ESR)
  Elias Rudberg,            KTH Stockholm,                Sweden      (DFT grid and basis info)
  Torgeir A. Ruden,         University of Oslo,           Norway      (Numerical derivatives in ABACUS)
  Kenneth Ruud,             UiT The Arctic U. of Norway,  Norway      (DALTON; ABACUS magnetic properties and much more)
  Pawel Salek,              KTH Stockholm,                Sweden      (DALTON; DFT code)
  Claire C. M. Samson       University of Karlsruhe       Germany     (Boys localization, r12 integrals in ERI)
  Alfredo Sanchez de Meras, University of Valencia,       Spain       (CC module, Cholesky decomposition)
  Trond Saue,               Paul Sabatier University,     France      (direct Fock matrix construction)
  Stephan P. A. Sauer,      University of Copenhagen,     Denmark     (SOPPA(CCSD), SOPPA prop., AOSOPPA, vibrational g-factors)
  Bernd Schimmelpfennig,    Forschungszentrum Karlsruhe,  Germany     (AMFI module)
  Anna K. Schnack-Petersen, University of Copenhagen,     Denmark     (RPA(D) and HRPA(D) properties in AOSOPPA)
  Kristian Sneskov,         Aarhus University,            Denmark     (Polarizable embedding, QM/MM)
  Arnfinn H. Steindal,      UiT The Arctic U. of Norway,  Norway      (parallel QM/MM, polarizable embedding)
  Casper Steinmann,         Aalborg University,           Denmark     (QFIT, polarizable embedding)
  K. O. Sylvester-Hvid,     University of Copenhagen,     Denmark     (MC-SCRF)
  Peter R. Taylor,          VLSCI/Univ. of Melbourne,     Australia   (Symmetry handling ABACUS, integral transformation)
  Andrew M. Teale,          University of Nottingham,     England     (DFT-AC, DFT-D)
  David P. Tew,             University of Bristol,        England     (CCSD(R12))
  Julien Toulouse,          Sorbonne Univ.,               France      (MC-srDFT)
  Olav Vahtras,             KTH Stockholm,                Sweden      (Triplet response, spin-orbit, ESR, TDDFT, open-shell DFT)
  Lucas Visscher,           Vrije Universiteit Amsterdam, Netherlands (Frozen density embedding)
  David J. Wilson,          La Trobe University,          Australia   (DFT Hessian and DFT magnetizabilities)
  Hans Agren,               KTH Stockholm,                Sweden      (SIRIUS module, RESPONS, MC-SCRF solvation)
 --------------------------------------------------------------------------------

     Date and time (Darwin) : Wed Sep  8 11:30:24 2021
     Host name              : dhcpu489.scarlan.ki.ku.dk               

 * Work memory size             :    64000000 =  488.28 megabytes.

 * Directories for basis set searches:
   1) /Users/aehb/Desktop/dalton/cp_test
   2) /Users/aehb/Desktop/dalton/build/basis


Compilation information
-----------------------

 Who compiled             | aehb
 Host                     | dhcpu489.scarlan.ki.ku.dk
 System                   | Darwin-20.6.0
 CMake generator          | Unix Makefiles
 Processor                | x86_64
 64-bit integers          | ON
 MPI                      | OFF
 Fortran compiler         | /usr/local/bin/gfortran-9
 Fortran compiler version | unknown
 C compiler               | /usr/local/bin/gcc-9
 C compiler version       | unknown
 C++ compiler             | /usr/local/bin/g++-9
 C++ compiler version     | unknown
 Static linking           | OFF
 Last Git revision        | b4891b7e7922fcabfb1332e479558cca56faa20d
 Git branch               | MP3_energy
 Configuration time       | 2021-09-08 11:19:19.437166


   Content of the .dal input file
 ----------------------------------

**DALTON
.RUN WAVE FUNCTIONS
**WAVE FUNCTIONS
.CC
*CC INP
.MP3
.FREEZE
1 0
**END OF


   Content of the .mol file
 ----------------------------

BASIS
6-31G
Test system for CPS(D) energy series.

Atomtypes=1 Nosymmetry Angstrom Charge=-1
Charge=9 Atoms=1
F   0.000000   0.000000        0.000000


       *******************************************************************
       *********** Output from DALTON general input processing ***********
       *******************************************************************

 --------------------------------------------------------------------------------
   Overall default print level:    0
   Print level for DALTON.STAT:    1

    HERMIT 1- and 2-electron integral sections will be executed
    "Old" integral transformation used (limited to max 255 basis functions)
    Wave function sections will be executed (SIRIUS module)
 --------------------------------------------------------------------------------


   ****************************************************************************
   *************** Output of molecule and basis set information ***************
   ****************************************************************************


    The two title cards from your ".mol" input:
    ------------------------------------------------------------------------
 1: Test system for CPS(D) energy series.                                   
 2:                                                                         
    ------------------------------------------------------------------------

  Coordinates are entered in Angstrom and converted to atomic units.
          - Conversion factor : 1 bohr = 0.52917721 A

  Atomic type no.    1
  --------------------
  Nuclear charge:   9.00000
  Number of symmetry independent centers:    1
  Number of basis sets to read;    2
  Basis set file used for this atomic type with Z =   9 :
     "/Users/aehb/Desktop/dalton/build/basis/6-31G"

  Info about the basis set file: your basis has no documentation.
  Basis set: 6-31G                                                                           


                         SYMGRP: Point group information
                         -------------------------------

@    Point group: C1 


                                 Isotopic Masses
                                 ---------------

                           F          18.998403

                       Total mass:    18.998403 amu
                       Natural abundance: 100.000 %

 Center-of-mass coordinates (a.u.):    0.000000    0.000000    0.000000
 Center-of-mass coordinates (Angs):    0.000000    0.000000    0.000000


  Atoms and basis sets
  --------------------

  Number of atom types :    1
  Total number of atoms:    1

  Basis set used is "6-31G" from the basis set library.

  label    atoms   charge   prim   cont     basis
  ----------------------------------------------------------------------
  F           1    9.0000    22     9      [10s4p|3s2p]                                       
  ----------------------------------------------------------------------
  total:      1    9.0000    22     9
  ----------------------------------------------------------------------

  Threshold for neglecting AO integrals:  1.00D-12


  Cartesian Coordinates (a.u.)
  ----------------------------

  Total number of coordinates:    3
  F       :     1  x   0.0000000000    2  y   0.0000000000    3  z   0.0000000000


@ This is an atomic calculation.


                     .---------------------------------------.
                     | Starting in Integral Section (HERMIT) |
                     `---------------------------------------'



 ***************************************************************************************
 ****************** Output from **INTEGRALS input processing (HERMIT) ******************
 ***************************************************************************************


 - Using defaults, no **INTEGRALS input found

 Default print level:        1

 Calculation of one- and two-electron Hamiltonian integrals.

 Center of mass  (bohr):      0.000000000000      0.000000000000      0.000000000000
 Operator center (bohr):      0.000000000000      0.000000000000      0.000000000000
 Gauge origin    (bohr):      0.000000000000      0.000000000000      0.000000000000
 Dipole origin   (bohr):      0.000000000000      0.000000000000      0.000000000000

Calculation of 2-electron integrals are postponed until they are needed.


     ************************************************************************
     ************************** Output from HERINT **************************
     ************************************************************************



                      Nuclear contribution to dipole moments
                      --------------------------------------

                 au               Debye          C m (/(10**-30)

      x      0.00000000         0.00000000         0.00000000
      y      0.00000000         0.00000000         0.00000000
      z      0.00000000         0.00000000         0.00000000


  Total CPU  time used in HERMIT:   0.00 seconds
  Total wall time used in HERMIT:   0.00 seconds


                        .----------------------------------.
                        | End of Integral Section (HERMIT) |
                        `----------------------------------'



                   .--------------------------------------------.
                   | Starting in Wave Function Section (SIRIUS) |
                   `--------------------------------------------'


 *** Output from Huckel module :

     Using EWMO model:          T
     Using EHT  model:          F
     Number of Huckel orbitals each symmetry:    5

 EWMO - Energy Weighted Maximum Overlap - is a Huckel type method,
        which normally is better than Extended Huckel Theory.
 Reference: Linderberg and Ohrn, Propagators in Quantum Chemistry (Wiley, 1973)

 Huckel EWMO eigenvalues for symmetry :  1
          -26.382700      -1.572500      -0.730000      -0.730000      -0.730000

 **********************************************************************
 *SIRIUS* a direct, restricted step, second order MCSCF program       *
 **********************************************************************

 
     Date and time (Darwin) : Wed Sep  8 11:30:24 2021
     Host name              : dhcpu489.scarlan.ki.ku.dk               

 Title lines from ".mol" input file:
     Test system for CPS(D) energy series.                                   
                                                                             

 Print level on unit LUPRI =   2 is   0
 Print level on unit LUW4  =   2 is   5

@    (Integral direct) CC calculation.

@    This is a combination run starting with
@              a restricted, closed shell Hartree-Fock calculation


 Initial molecular orbitals are obtained according to
 ".MOSTART EWMO  " input option

     Wave function specification
     ============================

     For the specification of the Coupled Cluster: see below.

@    Wave function type        --- CC ---
@    Number of closed shell electrons          10
@    Number of electrons in active shells       0
@    Total charge of the molecule              -1

@    Spin multiplicity and 2 M_S                1         0
@    Total number of symmetries                 1 (point group: C1 )
@    Reference state symmetry                   1 (irrep name : A  )

     Orbital specifications
     ======================
@    Abelian symmetry species          All |    1
@                                          |  A  
                                       --- |  ---
@    Occupied SCF orbitals               5 |    5
@    Secondary orbitals                  4 |    4
@    Total number of orbitals            9 |    9
@    Number of basis functions           9 |    9

     Optimization information
     ========================
@    Number of configurations                 1
@    Number of orbital rotations             20
     ------------------------------------------
@    Total number of variables               21

     Maximum number of Fock   iterations      0
     Maximum number of DIIS   iterations     60
     Maximum number of QC-SCF iterations     60
     Threshold for SCF convergence     1.00D-06


 Changes of defaults for CC:
 ---------------------------


 -Implicit frozen core calculation
  


 ***********************************************
 ***** DIIS acceleration of SCF iterations *****
 ***********************************************

 C1-DIIS algorithm; max error vectors =    8

 Iter      Total energy        Error norm    Delta(E)  DIIS dim.
 -----------------------------------------------------------------------------

Calculating AOSUPINT

---> (Re)generating AOTWOINT


     ************************************************************************
     ************************** Output from HERINT **************************
     ************************************************************************


 Threshold for neglecting two-electron integrals:  1.00D-12
 HERMIT - Number of two-electron integrals written:         213 ( 20.6% )
 HERMIT - Megabytes written:                              0.007

     (Precalculated AO two-electron integrals are transformed to P-supermatrix elements.
      Threshold for discarding integrals :  1.00D-12 )
@  1    -99.2722004625        1.45380D+00   -9.93D+01    1
      Virial theorem: -V/T =      1.970756
@    MULPOP   F      -1.00; 
   1  Level shift: doubly occupied orbital energies shifted by -2.00D-01
 -----------------------------------------------------------------------------
@  2    -99.3375344003        5.97727D-01   -6.53D-02    2
      Virial theorem: -V/T =      2.007614
@    MULPOP   F      -1.00; 
   2  Level shift: doubly occupied orbital energies shifted by -1.00D-01
 -----------------------------------------------------------------------------
@  3    -99.3501508181        2.69556D-02   -1.26D-02    3
      Virial theorem: -V/T =      1.997645
@    MULPOP   F      -1.00; 
   3  Level shift: doubly occupied orbital energies shifted by -2.50D-02
 -----------------------------------------------------------------------------
@  4    -99.3501804045        2.03431D-03   -2.96D-05    4
      Virial theorem: -V/T =      1.997193
@    MULPOP   F      -1.00; 
 -----------------------------------------------------------------------------
@  5    -99.3501805990        4.64243D-06   -1.94D-07    5
      Virial theorem: -V/T =      1.997213
@    MULPOP   F      -1.00; 
 -----------------------------------------------------------------------------
@  6    -99.3501805990        6.53046D-08   -4.41D-13    6

@ *** DIIS converged in   6 iterations !
@     Converged SCF energy, gradient:    -99.350180599030    6.53D-08
    - total time used in SIRFCK :              0.00 seconds


 *** SCF orbital energy analysis ***

 Number of electrons        :   10
 Closed shell orbitals      :    5

 Sym       Hartree-Fock orbital energies

1 A     -25.69352129    -0.96658051    -0.07774658    -0.07774658    -0.07774658
          1.99257023     1.99257023     1.99257023     2.14386110

    E(LUMO) :     1.99257023 au (symmetry 1)
  - E(HOMO) :    -0.07774658 au (symmetry 1)
  ------------------------------------------
    gap     :     2.07031681 au

 --- Writing SIRIFC interface file

--------------------------------------------------------------------------------


     .---------------------------------------------------------------.
     | HF-SCF KT IPs [a.u. & eV], pole strengths, and Dyson orbitals |
     `---------------------------------------------------------------'

     HF-SCF KT symmetry 1 ( A  ) in point group C1  -- 5 principal poles

   ============  ============  ============  ============  ============
      25.693521      0.966581      0.077747      0.077747      0.077747
     699.156279     26.301994      2.115592      2.115592      2.115592
   ------------  ------------  ------------  ------------  ------------
       1.000000      1.000000      1.000000      1.000000      1.000000
   ============  ============  ============  ============  ============
       1.000000      0.000000      0.000000      0.000000      0.000000
       0.000000     -1.000000      0.000000      0.000000      0.000000
       0.000000      0.000000     -1.000000      0.000000      0.000000
       0.000000      0.000000      0.000000      1.000000      0.000000
       0.000000      0.000000      0.000000      0.000000      1.000000
   ============  ============  ============  ============  ============
  

@ Summary of HF-SCF KT ionization potentials [eV] and pole strengths:

@ Symmetry 1 (A  ) :   699.156      26.302       2.116       2.116       2.116
@ Pole strengths   :     1.000       1.000       1.000       1.000       1.000

  === End of HF-SCF KT output ===
--------------------------------------------------------------------------------


 CPU and wall time for SCF :       0.005       0.006


                       .-----------------------------------.
                       | --- Final results from SIRIUS --- |
                       `-----------------------------------'


@    Spin multiplicity:           1
@    Spatial symmetry:            1 ( irrep  A   in C1  )
@    Total charge of molecule:   -1

@    Final HF energy:             -99.350180599030                 
@    Nuclear repulsion:             0.000000000000
@    Electronic energy:           -99.350180599030

@    Final gradient norm:           0.000000065305

 
     Date and time (Darwin) : Wed Sep  8 11:30:24 2021
     Host name              : dhcpu489.scarlan.ki.ku.dk               

File label for MO orbitals:   8Sep21   FOCKDIIS

 (Only coefficients > 0.0100 are printed.)

 Molecular orbitals for symmetry species 1  (A  )
 ------------------------------------------------

    Orbital         1        2        3        4        5        6        7
   1 F   :1s     0.9955  -0.2372   0.0000   0.0000   0.0000   0.0000   0.0000
   2 F   :1s     0.0205   0.4881   0.0000   0.0000   0.0000   0.0000   0.0000
   3 F   :1s    -0.0040   0.6020   0.0000   0.0000   0.0000   0.0000   0.0000
   4 F   :2px    0.0000   0.0000   0.6178   0.0000   0.0000   0.9751   0.0000
   5 F   :2py    0.0000   0.0000   0.0000   0.6178   0.0000   0.0000   0.0000
   6 F   :2pz    0.0000   0.0000   0.0000   0.0000   0.6178   0.0000   0.9751
   7 F   :2px    0.0000   0.0000   0.5361   0.0000   0.0000  -1.0223   0.0000
   8 F   :2py    0.0000   0.0000   0.0000   0.5361   0.0000   0.0000   0.0000
   9 F   :2pz    0.0000   0.0000   0.0000   0.0000   0.5361   0.0000  -1.0223

    Orbital         8        9
   1 F   :1s     0.0000   0.1295
   2 F   :1s     0.0000  -1.4959
   3 F   :1s     0.0000   1.4272
   5 F   :2py    0.9751   0.0000
   8 F   :2py   -1.0223   0.0000

  Total CPU  time used in SIRIUS :   0.01 seconds
  Total wall time used in SIRIUS :   0.01 seconds

 
     Date and time (Darwin) : Wed Sep  8 11:30:24 2021
     Host name              : dhcpu489.scarlan.ki.ku.dk               


                     .---------------------------------------.
                     | End of Wave Function Section (SIRIUS) |
                     `---------------------------------------'



                    .------------------------------------------.
                    | Starting in Coupled Cluster Section (CC) |
                    `------------------------------------------'

  

 *******************************************************************************
 *******************************************************************************
 *                                                                             *
 *                                                                             *
 *                    START OF COUPLED CLUSTER CALCULATION                     *
 *                                                                             *
 *                                                                             *
 *******************************************************************************
 *******************************************************************************


  I am freezing!
 Freezing HF-orbital   1 of symmetry   1 and with orbital energy  -25.6935
 In total frozen-core per symmetry-class:  1
 


 CCR12 ANSATZ =   0

 CCR12 APPROX =   0

   

 *******************************************************************
 *                                                                 *
 *----------                                             >---------*
 *---------- OUTPUT FROM COUPLED CLUSTER ENERGY PROGRAM  >---------*
 *----------                                             >---------*
 *                                                                 *
 *******************************************************************


             The Direct Coupled Cluster Energy Program
             -----------------------------------------


          Number of t1 amplitudes                 :        16
          Number of t2 amplitudes                 :       136
          Total number of amplitudes in ccsd      :       152



          Perturbation calculation - NO WAVEFUNCTION OPTIMIZATION
  

 *******************************************************************************
 *******************************************************************************
 *                                                                             *
 *                                                                             *
 *                           STARTING MP3 CALCULATION                          *
 *                                                                             *
 *                                                                             *
 *******************************************************************************
 *******************************************************************************



          Initializing calculation of first order amplitude corrections


          A wonder has occurred: We got the first order amplitude corrections!


          The second order correction to the energy is:  -0.1215125667


          Initializing calculation of second order amplitude corrections


          A wonder has occurred: We got the second order amplitude corrections!


          The third order correction to the energy is:   0.0019134786
  

 *******************************************************************************
 *******************************************************************************
 *                                                                             *
 *                                                                             *
 *                            FINSHED MP3 CALCULATION                          *
 *                                                                             *
 *                                                                             *
 *******************************************************************************
 *******************************************************************************

  

 *******************************************************************************
 *******************************************************************************
 *                                                                             *
 *                                                                             *
 *                   SUMMARY OF COUPLED CLUSTER CALCULATION                    *
 *                                                                             *
 *                                                                             *
 *******************************************************************************
 *******************************************************************************

                                                                                
            Total SCF   energy:                         -99.3501805990          
                                                                                
            MP2 correction:                              -0.1215125667          
                                                                                
            Total MP2 energy:                           -99.4716931657          
                                                                                
            MP3 correction:                               0.0019134786          
                                                                                
            Total MP3 energy:                           -99.4697796871          
  

 *******************************************************************************
 *******************************************************************************
 *                                                                             *
 *                                                                             *
 *                      END OF COUPLED CLUSTER CALCULATION                     *
 *                                                                             *
 *                                                                             *
 *******************************************************************************
 *******************************************************************************


 CPU and wall time for CC :       0.005       0.009

 
     Date and time (Darwin) : Wed Sep  8 11:30:24 2021
     Host name              : dhcpu489.scarlan.ki.ku.dk               


                      .-------------------------------------.
                      | End of Coupled Cluster Section (CC) |
                      `-------------------------------------'

  Total CPU  time used in DALTON:   0.02 seconds
  Total wall time used in DALTON:   0.03 seconds

 
     Date and time (Darwin) : Wed Sep  8 11:30:24 2021
     Host name              : dhcpu489.scarlan.ki.ku.dk               
